.isort.cfg

```diff
@@ -3,6 +3,6 @@ multi_line_output=3
 include_trailing_comma=True
 force_grid_wrap=0
 combine_as_imports=True
-line_length=80
+line_length=88
```

Comment:
...?

Reply:
I'm seeing more libraries default to 88 instead of 80 (including our code formatter black, for which 88 is the default). It just makes the code a lot more readable; a whole lot of lines don't need to wrap now with just those extra 8 characters.